openstack/blockstorage/extensions/volumeactions/results.go

```diff
@@ -50,6 +54,37 @@ func (r commonResult) Extract() (map[string]interface{}, error) {
 	return s.ConnectionInfo, err
 }
 
+// VolumeImage contains information about volume upload to an image service.
+type VolumeImage struct {
+	// The ID of a volume an image is created from.
+	VolumeId string `json:"id"`
+	// Container format, may be bare, ofv, ova, etc.
+	ContainerFormat string `json:"container_format"`
+	// Disk format, may be raw, qcow2, vhd, vdi, vmdk, etc.
+	DiskFormat string `json:"disk_format"`
+	// Human-readable description for the volume.
+	Description string `json:"display_description"`
+	// The ID of an image being created.
+	ImageId string `json:"image_id"`
+	// Human-readable display name for the image.
+	ImageName string `json:"image_name"`
+	// Size of the volume in GB.
+	Size int `json:"size"`
+	// Current status of the volume.
+	Status string `json:"status"`
+	// The date when this volume was last updated.
+	UpdatedAt time.Time `json:"-"`
```

Comment:
@vitkhab Also, in your structs, you will need to implement a custom unmarshaller for parsing the time using `UnmarshalJSON`. Please see how it's implemented in the `volumes` package as a reference.